### PR TITLE
How to Use MongoDB Atlas in API Integration Services

### PR DESCRIPTION
**Pull request by nraboy**
http://docs-devhub-staging.s3-website-us-east-1.amazonaws.com/7c25ac2/devhub/docsworker-xlarge/how-to-use-mongodb-atlas-in-api-integration-services/how-to/how-to-use-mongodb-atlas-in-api-integration-services

### CODE REVIEW

File: source/how-to/how-to-use-mongodb-atlas-in-api-integration-services.txt [hunk 1]
```diff
@@ -0,0 +1,390 @@
+:template: devhub-article
+ 
+.. meta-description::
+   MongoDB Atlas, our fully-managed database-as-a-service offering, was never meant to be used only by database administrators and analysts, it is also used by developers who might have tools of their own through which they’d like to integrate with the world instead of using development platforms directly. That’s why we developed a full-featured API for MongoDB Atlas. We have already covered how to work with MongoDB Atlas API from command line in a recent blog post. Now, it’s time to take advantage of it with 3rd-party tools that developers love and use today – often called API integration services, but also going by other monikers such as online workflows, serverless API frameworks, etc.
```
Comment (schmalliso) [inline source/how-to/how-to-use-mongodb-atlas-in-api-integration-services.txt]: This is probably too wordy for a meta-description (and also for the twitter description below). Meta-description should aim to be 125 - 155 characters. The twitter description is truncated to three lines in timeline and expanded Tweet, which apparently works out to about 200 characters.

File: source/how-to/how-to-use-mongodb-atlas-in-api-integration-services.txt [hunk 2]
```diff
@@ -0,0 +1,390 @@
...
+   workflow up to be triggered via a URL. You can use this URL in your
+   scripts or anywhere else that accepts a URL for a web hook.
+
+   Another interesting trigger to explore is the PagerDuty trigger. Using a
+   MongoDB Atlas integration with **PagerDuty**, you can have your Flow
+   execute automatically every time a PagerDuty alert goes out. This can
+   allow you to automate updating a cluster every time you get a low disk
+   space alert from PagerDuty, for example.
+
+   **Conclusion**
```
[the remark this answers is not in the document]
Comment (MaBeuLux88) [reply source/how-to/how-to-use-mongodb-atlas-in-api-integration-services.txt]: ..conclusion:: is a new directive? I thought it was ..summary:: ?

[the remark this answers is not in the document]
Comment (schmalliso) [reply source/how-to/how-to-use-mongodb-atlas-in-api-integration-services.txt]: You're right, summary is correct. I have looked at too many PRs today :D

File: source/includes/authors/savateyev-aleksey.rst [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.. author:: 
+   :name: Aleksey Savateyev
+   
+   TBD
```
Comment (MaBeuLux88) [inline source/includes/authors/savateyev-aleksey.rst]: Ask for a bio + picture if you can.

File: source/how-to/how-to-use-mongodb-atlas-in-api-integration-services.txt [hunk 4]
```diff
@@ -0,0 +1,390 @@
...
+
+   We have partnered with a number of vendors in this space to allow users
+   to take advantage of Atlas and MongoDB features from within their
+   favored development environments. This blog will provide overviews and
+   walkthroughs of 2 integrations with API integration platforms:
+   `Stamplay <http://www.stamplay.com/>`_ and
+   `Built.io <http://built.io/?utm_source=mongodb&utm_campaign=mongodb%20workflow%20blog&utm_medium=blog&utm_content=homepage>`_.
+
+   **Getting started with MongoDB Atlas**
+   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (MaBeuLux88) [inline source/how-to/how-to-use-mongodb-atlas-in-api-integration-services.txt]: Wrong level of header. Please be consistent with what the header definitions : https://wiki.corp.mongodb.com/display/DE/ReStructured+Text+Cheat+Sheet

File: source/how-to/how-to-use-mongodb-atlas-in-api-integration-services.txt [hunk 5]
```diff
@@ -0,0 +1,390 @@
...
+.. twitter::
+   :site: @mongodb
+   :title: "How to Use MongoDB Atlas in API Integration Services"
+   :image: https://webassets.mongodb.com/_com_assets/cms/background-header-o1ut2fuv7m.png
+ 
+   MongoDB Atlas, our fully-managed database-as-a-service offering, was never meant to be used only by database administrators and analysts, it is also used by developers who might have tools of their own through which they’d like to integrate with the world instead of using development platforms directly. That’s why we developed a full-featured API for MongoDB Atlas. We have already covered how to work with MongoDB Atlas API from command line in a recent blog post. Now, it’s time to take advantage of it with 3rd-party tools that developers love and use today – often called API integration services, but also going by other monikers such as online workflows, serverless API frameworks, etc.
+ 
+.. tags::
+ 
+   * cloud
```
Comment (MaBeuLux88) [inline source/how-to/how-to-use-mongodb-atlas-in-api-integration-services.txt]: Change to Cloud with an uppercase C.

File: source/how-to/how-to-use-mongodb-atlas-in-api-integration-services.txt [hunk 6]
```diff
@@ -0,0 +1,390 @@
...
+.. include:: /includes/authors/savateyev-aleksey.rst
+ 
+.. pubdate:: January 19, 2017
+ 
+.. type:: how-to
+.. level:: beginner
+ 
+.. products::
+ 
+   * MongoDB Atlas
```
Comment (MaBeuLux88) [inline source/how-to/how-to-use-mongodb-atlas-in-api-integration-services.txt]: Change to "Atlas".
Else you will fail the sanity-check.py

File: source/how-to/how-to-use-mongodb-atlas-in-api-integration-services.txt [hunk 7]
```diff
@@ -0,0 +1,390 @@
+:template: devhub-article
```
Comment (MaBeuLux88) [inline source/how-to/how-to-use-mongodb-atlas-in-api-integration-services.txt]: Rename this file "atlas-api-integration-services.txt" ==> Better for SEO.

File: source/how-to/how-to-use-mongodb-atlas-in-api-integration-services.txt [hunk 8]
```diff
@@ -0,0 +1,390 @@
...
+   and sophisticated applications.
+
+   Want to know more about MongoDB Atlas? Check out `MongoDB Atlas API
+   Documentation <https://docs.atlas.mongodb.com/reference/api-resources/>`_
+   and `M034 course at MongoDB
+   University <https://university.mongodb.com/courses/M034/about>`_!
+
+   --------------
+
+   *About the Author – Aleksey Savateyev*
```
[the remark this answers is not in the document]
Comment (nraboy) [reply source/how-to/how-to-use-mongodb-atlas-in-api-integration-services.txt]: Updated the bio file in latest push
